src/Concordium/Client/Runner.hs

```diff
@@ -3035,22 +3119,22 @@ processBakerConfigureCmd baseCfgDir verbose backend txOpts isBakerConfigure cbCa
             unless confirmed exitTransactionCancelled
 
     warnIfCapitalIsLowered capital stakedAmount = do
-        cooldownDate <- withClient backend $ do
-            bcpRes <- getBlockChainParameters Best
-            case getResponseValue bcpRes of
-                Left (_, err) -> do
-                    logError ["Could not get the validator cooldown period: " <> err]
-                    exitTransactionCancelled
-                Right v -> getBakerCooldown v
         when (capital < stakedAmount) $ do
+            cooldownDate <- withClient backend $ do
+                bcpRes <- getBlockChainParameters Best
+                case getResponseValue bcpRes of
+                    Left (_, err) -> do
+                        logError ["Could not get the validator cooldown period: " <> err]
+                        exitTransactionCancelled
+                    Right v -> getBakerCooldown v
             let removing = capital == 0
             if removing
                 then logWarn ["This will remove the validator."]
                 else logWarn ["The new staked value appears to be lower than the amount currently staked on chain by this validator."]
             let decreaseOrRemove = if removing then "Removing a validator" else "Decreasing the amount a validator is staking"
             logWarn [decreaseOrRemove ++ " will lock the stake of the validator for a cooldown period before the CCD are made available."]
             logWarn ["During this period it is not possible to update the validator's stake, or stop the validator."]
-            logWarn [[i|The current validator cooldown would last until approximately #{cooldownDate}|]]
+            logWarn [[i|The validator cooldown will last until approximately #{cooldownDate}|]]
```

Comment:
Out of curiosity, why do we say "approximately" here? Is it not a precise time? Why might it vary?

Reply:
The actual timestamp of the payday block (which is when the amount actually leaves cooldown) is not precisely determined in advance. Typically, it will be the third block whose timestamp is after the (payday) epoch transition time.